CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 
 All notable changes to this project are documented in this file.
 
+## 0.12.0
+
+**Release date:** 2021-04-21
+
+This prerelease comes with support for SSH keys with a passphrase.
```

Comment:
There are two breaking changes that should be mentioned:

* `.sourceignore` are now loaded by traversing through a directory tree, instead of just looking at the root.
* `ValuesFile` (string) in the `HelmRelease` has been replaced with `ValuesFiles` list, but is backwards compatible.